new_platforms/vsextension/ProjectWizard/ImportEnclaveCommand.cs

```diff
@@ -321,10 +356,12 @@ private async void Execute(object sender, EventArgs e)
                         project,
                         packageVersions);
 
+                    bool isWindows = (vcProject.keyword != "Linux");
+
                     // Add any configurations/platforms to the project.
                     AddConfiguration(project, "OPTEE-Simulation-Debug", "Debug");
```

Comment:
Should we avoid anything OP-TEE-related on Linux for now?

Reply:
Done